### PR TITLE
Backfill Lambda changelog for 3.2.1

### PR DESCRIPTION
**Pull request by akarve**

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -23,10 +24,27 @@
 * [Added] Add `--meta` argument to `push` ([#1793](https://github.com/quiltdata/quilt/issues/1793))
 * [Fixed] Fix crash in `list-packages` ([#1852](https://github.com/quiltdata/quilt/issues/1852))
 
+## Catalog, Lambdas
+* [Added] Ability to preview larger Jupyter notebooks; warning when cells are
+elided ([#1823](https://github.com/quiltdata/quilt/issues/1823),
```
Comment (sir-sigurd) [inline docs/CHANGELOG.md:29]: Unfortunately GitBook renders each line in a list item as a separate paragraph like here:
https://docs.quiltdata.com/#quilt-in-action

Not sure if there is a way to fix it and if we should fix it. Instead we might want to try to switch to [readthedocs.io](https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx).

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -23,10 +24,27 @@
...
+* [Fixed] Bugs involving bad or missing package stats during S3 Select calls
+([#1829](https://github.com/quiltdata/quilt/issues/1829))
+* [Added] Object size to package browsing experience in catalog
+([#1744](https://github.com/quiltdata/quilt/issues/1744))
+* [Added] Total number of packages to catalog Overview tab
+([#1808](https://github.com/quiltdata/quilt/issues/1808))
+* [Fixed] Overly aggressive 40X retry logic in es/indexer
+([#1804](https://github.com/quiltdata/quilt/issues/1804))
+* [Fixed] Semantic bugs in ElasticSearch timeouts (queries now time out properly)
+([#1801](https://github.com/quiltdata/quilt/issues/1801))
```
Comment (sir-sigurd) [inline docs/CHANGELOG.md:41]: #1801 is a wrong PR.

**Review by sir-sigurd — CHANGES_REQUESTED** [review]
We should group items by their type (added, fixed).

File: docs/CHANGELOG.md (hunk 1 above)
Comment (akarve) [reply docs/CHANGELOG.md]: This is silly, and a bit of Google Fu finds no workarounds, but I think we should live with it for now.

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -21,12 +22,22 @@
 
 ## CLI
 * [Added] Add `--meta` argument to `push` ([#1793](https://github.com/quiltdata/quilt/issues/1793))
+
+## Catalog, Lambdas
+* [Added] Ability to preview larger Jupyter notebooks; warning when cells are elided ([#1823](https://github.com/quiltdata/quilt/issues/1823), [#1822](https://github.com/quiltdata/quilt/issues/1822))
+* [Added] Object size to package browsing experience in catalog ([#1744](https://github.com/quiltdata/quilt/issues/1744))
+* [Added] Total number of packages to catalog Overview tab ([#1808](https://github.com/quiltdata/quilt/issues/1808))
+* [Added] PDF and other file formats, including .gz ones, to catalog Overview tab
 * [Fixed] Fix crash in `list-packages` ([#1852](https://github.com/quiltdata/quilt/issues/1852))
```
Comment (sir-sigurd) [inline docs/CHANGELOG.md:31]: This one is still for CLI :slightly_smiling_face:

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -23,10 +24,20 @@
...
+* [Fixed] Bugs involving bad or missing package stats during S3 Select calls ([#1829](https://github.com/quiltdata/quilt/issues/1829))
+* [Fixed] Overly aggressive 40X retry logic in es/indexer ([#1804](https://github.com/quiltdata/quilt/issues/1804))
+* [Fixed] Semantic bugs in ElasticSearch timeouts (queries now time out properly) ([#1803](https://github.com/quiltdata/quilt/issues/1803))
+* [Fixed] Missing Helvetica issues for PDFs ([#1792](https://github.com/quiltdata/quilt/issues/1792))
+
 # 3.2.0 - 2020-09-08 - Package Registry Refactor
 ## Python:
 * Refactors local and s3 storage-layer code around a new PackageRegistry base class (to support improved file layouts in future releases)
-* Multithreaded download for large files, large performance gains when installing packages with large files, especially on large instances
+* Multi-threaded download for large files, large performance gains when installing packages with large files, especially on large instances
```
Comment (akarve) [inline docs/CHANGELOG.md:40]:
```suggestion
* Multi-threaded download for large files, 2X to 5X performance gains when installing packages with large files, especially on larger EC2 instances
```

**Review by sir-sigurd — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -4,9 +4,10 @@
 
 ## CLI
 
-## Catalog
+## Catalog, Lambdas
```
Comment (sir-sigurd) [inline docs/CHANGELOG.md:7]: Let's make this change in the separate commit/PR, because ideally we should this fix for 3.2.1 to stable branch.

**Review by sir-sigurd — CHANGES_REQUESTED** [review]
